db/re/item_db_equip.yml

```
@@ -145083,6 +145861,42 @@ Body:
       if (.@r>=7) {
          bonus2 bMagicAtkEle,Ele_All,5;
       }
+  - Id: 470061
+    AegisName: Cylinder_Boots_
+    Name: Cylinder Boots
+    Type: Armor
+    Weight: 400
+    Defense: 12
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      .@val = getskilllv("GN_CARTBOOST");
```

If I'm not wrong `精錬 : 可 破損 : しない`
```suggestion
      .@val = getskilllv("GN_CARTBOOST");
      bonus bUnbreakableShoes;
```

---

OK, Thanks!